plugins/tech-insights-backend/api-report.md

```diff
@@ -77,6 +82,12 @@ export interface FactRetrieverRegistry {
   register(registration: FactRetrieverRegistration): Promise<void>;
 }
 
+// @public
+export const initializePersistenceContext: (
+  database: PluginDatabaseManager,
+  options?: CreateDatabaseOptions,
```

Comment:
I wonder if it's worth renaming this to `InitializePersistenceContextOptions` or something similarly named as these could potentially be expanded with other options that's not Database specific?

Reply:
Renamed that as `PersistenceContextOptions`